tests/conftest.py

```diff
@@ -103,6 +103,12 @@ def fn(sources_dict):
     return fn
 
 
+# for tests which just need an input bundle, doesn't matter what it is
+@pytest.fixture
+def dummy_input_bundle():
```

Comment:
hmm interesting. well, `InputBundle()` is actually not fully functioning since it's a base class, it will raise exceptions if you try to use it: https://github.com/charles-cooper/vyper/blob/74c630ece2723cf24418acf9c635418ab2c2a461/vyper/compiler/input_bundle.py#L73-L77

so i wanted to use it for tests which need an object of type InputBundle, but can never actually call into it.